web/src/javascript/game.js

```diff
@@ -1,18 +1,15 @@
 import {
-  drawCellsOnMap,
+  drawFoodOnMap,
   drawMap,
   drawMiniMap,
```

Comment:
We will need to draw the mini-Map before drawing the food so that the food isnt on the mini-map, you can either change it here or make an Issue, idk if it takes long.